scanomatic/image_analysis/first_pass_image.py

```diff
@@ -423,9 +425,14 @@ def get_plate_im_section(
     def analyse_grayscale(self) -> None:
         current_model = self["current"].model
         try:
-            current_model.grayscale.section_values = detect_grayscale(
+            grayscale_im = get_grayscale_im_section(
                 self.im,
                 current_model.grayscale,
+            )
+            grayscale_config = get_grayscale(current_model.grayscale.name)
+            current_model.grayscale.section_values = detect_grayscale(
```

Comment:
just a curious question, we're only saving the `1` index, is this always the case from detect_grayscale? In that case perhaps nice to simplify the return type.

Reply:
Yes! It is always the case. But it is however a bit strange, it means that we are not using the `grayscale_segment_centers` from `grayscale_detection.find_grayscale_locations` anywhere in the code. That seems odd?

Reply:
It might not be as odd as it sounds, I probably used them during the tweaking of the detection algorithm to be able to visually inspect where they ended up and they've just remained there in case they became useful another day. I'd say if they're not used we can safely remove them and should we get to rewrite grayscale detection (#33) then we probably will need to start from scratch anyhow.